packages/salesforcedx-utils-vscode/src/types/index.ts

```diff
@@ -88,6 +88,7 @@ export type LocalComponent = DirFileNameSelection & {
   suffix?: string;
 };
 
-export type FunctionInfo = DirFileNameSelection & {
+export type FunctionInfo = {
+  fileName: string;
```

Comment:
Why don't we need the directory path anymore?

Reply:
Ok, so generateFunction takes language and figures out the path itself it looks like.

Reply:
I think previously we used `DirFileNameSelection` which provided `outputDir` in order to satisfy the requirements of `BaseTemplateCommand`, but it was never actually set.